kuksa-client/kuksa_client/grpc/__init__.py

```diff
@@ -457,7 +457,7 @@ def __init__(
         certificate_chain: Optional[Path] = None,
         *,
         ensure_startup_connection: bool = True,
-    ):
+):
```

Comment:
to much :(

Don't you have an automatic formatter in your IDE?

Reply:
Now I have